packages/cli/src/oclif/commands/team/add.js

```diff
@@ -66,16 +69,18 @@ TeamAddCommand.args = [
 TeamAddCommand.flags = buildFlags();
 TeamAddCommand.description = `Add a team member to your integration.
 
-These users come in two levels:
+These users come in three levels:
 
   * \`admin\`, who can edit everything about the integration
-  * \`subscriber\`, who can't directly access the app, but will receive periodic email updates. These updates include quarterly health socores and more.
+  * \`subscriber\`, who can't directly access the app, but will receive periodic email updates. These updates include quarterly health scores and more.
+  * \`collaborator\`, who has read-only access for the app, but will receive periodic email updates. These updates include quarterly health scores and more.
```

Comment:
As we're deprecating `subscriber` and `collaborator` is technically a superset of `subscriber`, maybe it makes more sense to move `collaborator` before `subscriber`? The same applies to `TeamAddCommand.args` and `TeamAddCommand.examples` as well.

On the help text, the "but" should probably be an "and":

```suggestion
  * \`collaborator\`, who has read-only access for the app, and will receive periodic email updates. These updates include quarterly health scores and more.
```

Reply:
Makes sense!